man/dbxincluder.txt

```diff
@@ -0,0 +1,24 @@
+NAME
```

Comment:
I could, but when packaging I don't want to require the full sphinx stack to build a man page.
Also, the man page is not intended to be included in the main documentation, rather as a specialized explanation of the command line tool.